lib/Redis.php

```diff
@@ -1752,7 +1752,7 @@ public function time() {
      * @yield mixed
      */
     public function evalSha($sha1, $keys = [], $args = []) {
-        return $this->send(array_merge(["evalsha"], $sha1, sizeof((array) $keys), (array) $keys, (array) $args));
+        return $this->send(array_merge(["evalsha"], $sha1, (array) sizeof((array) $keys), (array) $keys, (array) $args));
```

Comment:
I think this should be `[sizeof((array) $keys)]` instead.

Reply:
yes, it's better

Reply:
@vicla31 Could you make that change?

Reply:
@vicla31 ping
@kelunik please merge this